### ISSUE TITLE
Devportal API Comments page accessibility issues

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#8380]
### Describe your problem(s)

1. Delete buttons Text does not meet minimum AA contrast ratio (4.5:1).

![image](https://user-images.githubusercontent.com/20179540/84925507-a1666e00-b0e7-11ea-91f0-85fc6d084c11.png)

2. delete conformation modal should have aria-alertdialog

![image](https://user-images.githubusercontent.com/20179540/84928315-d4126580-b0eb-11ea-9848-57fe8b66332d.png)

3. yes button Text does not meet minimum AA contrast ratio (4.5:1).

4. delete modal title and description should be inline with the [HIG guidelines ](http://wso2.github.io/carbon-apimgt/HIG/#dialogs)

### How will you implement it

---
